apply_test.go

```diff
@@ -129,7 +128,58 @@ func TestCorruptPatch(t *testing.T) {
 	defer cleanup(fName)
 	writeOldFile(fName, t)
 
-	badPatch := []byte{0x44, 0x38, 0x86, 0x3c, 0x4f, 0x8d, 0x26, 0x54, 0xb, 0x11, 0xce, 0xfe, 0xc1, 0xc0, 0xf8, 0x31, 0x38, 0xa0, 0x12, 0x1a, 0xa2, 0x57, 0x2a, 0xe1, 0x3a, 0x48, 0x62, 0x40, 0x2b, 0x81, 0x12, 0xb1, 0x21, 0xa5, 0x16, 0xed, 0x73, 0xd6, 0x54, 0x84, 0x29, 0xa6, 0xd6, 0xb2, 0x1b, 0xfb, 0xe6, 0xbe, 0x7b, 0x70}
+	badPatch := []byte{
+		0x44,
+		0x38,
+		0x86,
+		0x3c,
+		0x4f,
+		0x8d,
+		0x26,
+		0x54,
+		0xb,
+		0x11,
+		0xce,
+		0xfe,
+		0xc1,
+		0xc0,
+		0xf8,
+		0x31,
+		0x38,
+		0xa0,
+		0x12,
+		0x1a,
+		0xa2,
+		0x57,
+		0x2a,
+		0xe1,
+		0x3a,
+		0x48,
+		0x62,
+		0x40,
+		0x2b,
+		0x81,
+		0x12,
+		0xb1,
+		0x21,
+		0xa5,
+		0x16,
+		0xed,
+		0x73,
+		0xd6,
+		0x54,
+		0x84,
+		0x29,
+		0xa6,
+		0xd6,
+		0xb2,
+		0x1b,
+		0xfb,
+		0xe6,
+		0xbe,
+		0x7b,
+		0x70,
+	}
```

Comment:
This change doesn't seem necessary.

Did you apply gofumpt on files?

I'm curious how you get this

Reply:
> Did you apply gofumpt on files?

I honestly don't know. I also had golines enabled in my editor, so that's probably what it was.

Reply:
As you may update your PR for [this](https://github.com/minio/selfupdate/pull/15#discussion_r1831664761), you could also revert this